mission_control/navigator_missions/navigator_missions/entrance_gate.py

```diff
@@ -0,0 +1,463 @@
+#!/usr/bin/env python
+from txros import util
+from navigator_missions.navigator import Navigator
+import numpy as np
+from mil_tools import rosmsg_to_numpy
+from twisted.internet import defer
+import math
+from mil_misc_tools import ThrowingArgumentParser
+from argparse import ArgumentTypeError
+import tf2_ros
+import rospy
```

Comment:
can't rospy in txros program

Reply:
Fixed https://github.com/jaxnb/NaviGator/commit/e26de47ddb4335edd1105f6102eb5d21daf7e730